src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateTableCommand.java

```diff
@@ -13,9 +13,7 @@
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-// TODO, hide table feature detail before it goes public,
-// https://github.com/stargate/data-api/pull/1360
-// @Schema(description = "Command that creates an api table.")
+@Schema(description = "Command that creates an api table.")
```

Comment:
Question: should we use capitalization like "API  table"? (usually acronyms are capitalized)

Reply:
Oh missed this comment. Will create a separate issue for this to track .https://github.com/stargate/data-api/issues/1712